sass/Modules/_buttom-see-more.scss

```diff
@@ -0,0 +1,19 @@
+#botton_SeeMore {
```

Comment:
this should be an atom, so move it to the atoms folder
maybe something like .cta should be fine to use as a class instead of that ID